lib/app/components/organisms/received_message.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:favoritism_communication/app/components/atoms/atoms.dart';
+import 'package:favoritism_communication/app/styles/styles.dart';
+
+import 'package:flutter/material.dart';
+
+class ReceivedMessage extends StatelessWidget {
+  const ReceivedMessage({
+    super.key,
+    required this.profileImage,
+    required this.senderName,
+    required this.text,
+  });
+
+  final String? profileImage;
+  final String senderName;
+  final String text;
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
```

Comment:
```suggestion
    return Row(
     crossAxisAlignment: CrossAxisAlignment.end,
```
CircleAvatorが上よりだとメッセージと位置が微妙に見えるので
下寄りにしましょう